packages/turf-standard-deviational-ellipse/index.js

```diff
@@ -0,0 +1,133 @@
+import { coordAll, featureEach } from '@turf/meta';
+import { getCoords } from '@turf/invariant';
+import { featureCollection, isObject, isNumber } from '@turf/helpers';
+import centerMean from '@turf/center-mean';
+import pointsWithinPolygon from '@turf/points-within-polygon';
+import ellipse from '@turf/ellipse';
+
+/**
+ * Takes a {@link FeatureCollection} and returns a standard deviational ellipse,
+ * also known as a “directional distribution.” The standard deviational ellipse
+ * aims to show the direction and the distribution of a dataset by drawing
+ * an ellipse that contains about one standard deviation’s worth (~ 70%) of the
+ * data.
+ *
+ * This module was created in consultation with the following articles:
+ *
+ * • Robert S. Yuill, “The Standard Deviational Ellipse; An Updated Tool for
+ * Spatial Description,” _Geografiska Annaler_ 53, no. 1 (1971): 28–39,
+ * doi:{@link https://doi.org/10.2307/490885|10.2307/490885}.
+ *
+ * • Paul Hanly Furfey, “A Note on Lefever’s “Standard Deviational Ellipse,”
+ * _American Journal of Sociology_ 33, no. 1 (1927): 94—98,
+ * doi:{@link https://doi.org/10.1086/214336|10.1086/214336}.
+ *
+ * It mirrors the functionality of {@link http://desktop.arcgis.com/en/arcmap/10.3/tools/spatial-statistics-toolbox/directional-distribution.htm|Directional Distribution} in ArcGIS and the {@link http://arken.nmbu.no/~havatv/gis/qgisplugins/SDEllipse/|QGIS Standard Deviational Ellipse Plugin}.
+ *
+ * @name standardDeviationalEllipse
+ * @param {Geometry|FeatureCollection<Point>} points GeoJSON points
```

Comment:
I think it would be best to drop Geometry Support and only allow `FeatureCollection<Point>`